src/emailservice/email_server.rb

```diff
@@ -12,11 +12,40 @@
 
 post "/send_order_confirmation" do
   data = JSON.parse(request.body.read, object_class: OpenStruct)
-  Pony.mail(
-    to:       data.email,
-    from:     "noreply@example.com",
-    subject:  "Your confirmation email",
-    body:     erb(:confirmation, locals: { order: data.order }),
-    via:      :logger
-  )
+
+  # get the current auto-instrumented span
+  current_span = OpenTelemetry::Trace.current_span
+  current_span.add_attributes({
+    "app.order.id" => data.order.order_id,
+    "app.shipping.tracking.id" => data.order.shipping_tracking_id,
+    "app.shipping.cost.currency" => data.order.shipping_cost.currency_code,
+    "app.shipping.cost" => data.order.shipping_cost.units.to_s + "." + 
```

Comment:
We should note down all the custom attributes we add to services somewhere :)

Reply:
I agree, any suggestions on where @mic-max ?

Reply:
We can start with a custom-instrumentation.md file in the docs folder. I imagine a table for each type of instrumentation (traces, metrics, logs). The traces table would contain a list of the attributes, a short description, and which service(s) you can find it in.  The metrics table would contain a list of custom metrics with descriptions and services found.